azext_iot/common/utility.py

```diff
@@ -597,3 +598,12 @@ def ensure_azure_namespace_path():
         sys.path.insert(0, ext_path)
 
     return
+
+
+def check_connection_string(connection_string):
```

Comment:
This is a generic function name for a very specific check. Also thinking its placement should be somewhere closer to device messaging modules vs in the common utility module.

Reply:
will move it to the mqtt module for now

Reply:
moved